src/containers/ipfs/useGetIpfsContentHook.ts

```diff
@@ -82,16 +93,17 @@ export const getTypeContent = async (dataCid, cid) => {
   return response;
 };
 
-const useGetIpfsContent = (cid, nodeIpfs) => {
+const useGetIpfsContent = (cid: string, nodeIpfs: IPFS) => {
   const [content, setContent] = useState('');
   const [text, setText] = useState(cid);
   const [typeContent, setTypeContent] = useState('');
   const [status, setStatus] = useState('understandingState');
   const [link, setLink] = useState(`/ipfs/${cid}`);
-  const [gateway, setGateway] = useState(null);
+  const [gateway, setGateway] = useState(false);
```

Comment:
why it should `false`? null looks ok

Reply:
no need to mix bool and null without any special case

Reply:
I don't really agree, lets discuss in future